content/en/platform/corda/5.0/application-networks/creating/notaries.md

```diff
@@ -10,15 +10,15 @@ menu:
 section_menu: corda5
 ---
 # Onboarding Notaries
-This section describes how to onboard a new member as a notary service representative. It assumes that you have configured the [MGM for the network]({{< relref "./mgm/_index.md" >}}). Onboarding a notary member is similar to any other member, but with the exceptions outlined on this page.
+This section describes how to onboard a new member as a {{< tooltip >}}notary{{< /tooltip >}} service representative. It assumes that you have configured the [MGM for the network]({{< relref "./mgm/_index.md" >}}). Onboarding a notary member is similar to any other member, but with the exceptions outlined on this page.
 
 {{< note >}}
-When onboarding a notary, you need to use the notary CPK to build a notary CPI.
+When onboarding a notary, you need to use the notary CPK to build a notary {{< tooltip >}}CPI{{< /tooltip >}}.
```

Comment:
This one still is playing up on my preview: 
<img width="1402" alt="image" src="https://github.com/corda/corda-docs-portal/assets/135221026/d3d45aa5-5dac-4b18-a640-e0d08673fee6">